packages/story-editor/src/components/errorBoundary/index.js

```diff
@@ -46,15 +47,16 @@ class ErrorBoundary extends Component {
   render() {
     const { error, errorInfo } = this.state;
     if (error) {
-      return <ErrorActions error={error} errorInfo={errorInfo} />;
+      return (
+        <ErrorActions
+          withStoryData={this.props.withStoryData}
+          error={error}
+          errorInfo={errorInfo}
+        />
+      );
     }
     return this.props.children;
   }
 }
 
-const shouldDisableErrorBoundaries =
-  typeof WEB_STORIES_DISABLE_ERROR_BOUNDARIES !== 'undefined' &&
-  WEB_STORIES_DISABLE_ERROR_BOUNDARIES === 'true';
-export default shouldDisableErrorBoundaries
-  ? ({ children }) => children
-  : ErrorBoundary;
```

Comment:
Why was this `WEB_STORIES_DISABLE_ERROR_BOUNDARIES` check removed?

If we wanna remove it, we should do so in a separate PR since this is referenced in many places. Otherwise we end up with dead code.

Reply:
Bad commit on my end --- at one point I created a new error boundary file